.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Build Firedrake
         run: |
           cd ..
-          ./firedrake/scripts/firedrake-install $COMPLEX --venv-name build --tinyasm --disable-ssh --minimal-petsc --slepc --documentation-dependencies --install thetis --install gusto --install icepack --install irksome --install femlium --no-package-manager || (cat firedrake-install.log && /bin/false)
+          ./firedrake/scripts/firedrake-install $COMPLEX --venv-name build --tinyasm --disable-ssh --minimal-petsc --slepc --documentation-dependencies --install thetis --install gusto --install icepack --install irksome --install femlium --no-package-manager --package_branch ufl merge_fenics || (cat firedrake-install.log && /bin/false)
```

Comment:
This is failing because you've written `--package_branch` instead of `--package-branch` (`-` rather than `_`)

Reply:
Yeah I was expecting tests to fail because master is broken so didn't check. Thanks!